sos/plugins/tomcat.py

```diff
@@ -22,19 +23,39 @@ class Tomcat(Plugin, RedHatPlugin):
     plugin_name = 'tomcat'
     profiles = ('webserver', 'java', 'services', 'sysmgmt')
 
-    packages = ('tomcat6', 'tomcat')
+    packages = ('tomcat', 'tomcat6', 'tomcat7', 'tomcat8')
 
     def setup(self):
         self.add_copy_spec([
             "/etc/tomcat",
-            "/etc/tomcat6"
+            "/etc/tomcat6",
+            "/etc/tomcat7",
+            "/etc/tomcat8"
         ])
 
         limit = self.get_option("log_size")
-        log_glob = "/var/log/tomcat*/catalina.out"
-        self.add_copy_spec_limit(log_glob, sizelimit=limit)
+
+        if not self.get_option("all_logs"):
+            log_glob = "/var/log/tomcat*/catalina.out"
+            self.add_copy_spec_limit(log_glob, sizelimit=limit)
+
+            # get today's date in iso format so that days/months below 10
+            # prepend 0
+            today = datetime.date(datetime.now()).isoformat()
+            log_glob = "/var/log/tomcat*/catalina.%s.log" % today
+            self.add_copy_spec_limit(log_glob, sizelimit=limit)
```

Comment:
Thanks - this looks good & is very similar to other plugins. I'd like to streamline some of this in the future (across all plugins) but I think this gives us a reasonable lot of data for the time being - I'd not realised how little we captured on recent versions - I'm surprised nobody complained before!